DigitalLearningSolutions.Data.Tests/NBuilderHelpers/NBuilderAlphabeticalPropertyNamingHelper.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.NBuilderHelpers
+{
+    using System;
+    using NUnit.Framework;
+
+    public static class NBuilderAlphabeticalPropertyNamingHelper
+    {
+        public static string IndexToAlphabeticalString(int index)
+        {
+            if (index < 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(index) , @"Index must be greater than or equal to zero");
+            }
+
+            var remainder = index % 26;
+            var place = Convert.ToInt32(Math.Floor(index / 26.0));
+
+            var s = new string('Z', place);
+            s += (char)(remainder + 65);

```

Comment:
I don't think many people would know offhand what this does - could you wrap it in a oneline method called something like `GetNthLetterOfAlphabet(int n)` so that it's more obvious and people will feel more confident using this method?